config/roguelike_dungeons/settings/loot_weapons.json

```diff
@@ -0,0 +1,148 @@
+		{"data": {"name": "TConstruct:handGuard", "meta": 16}, "weight": 1},
+		{"data": {"name": "TConstruct:handGuard", "meta": 6}, "weight": 1},
+		{"data": {"name": "TConstruct:handGuard", "meta": 1647}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 16}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 15}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 1647}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 16}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 15}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 1647}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 16}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 15}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 1647}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 16}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 15}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 1647}, "weight": 1}
+      ]
+    },
+	{"level": 4, "type": "WEAPONS", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "TConstruct:SwordBlade", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:SwordBlade", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:SwordBlade", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:wideGuard", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:wideGuard", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:wideGuard", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:handGuard", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:handGuard", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:handGuard", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:arrowhead", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:largeSwordBlade", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:BowLimbPart", "meta": 10}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 201}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 11}, "weight": 1},
+		{"data": {"name": "TConstruct:CrossbowLimbPart", "meta": 10}, "weight": 1}
+      ]
+    },
+      {"level": 4, "type": "WEAPONS", "each": true, "quantity": 1, "loot": [
+		{"data": {"name": "GalaxySpace:item.QuantBow", "meta": 0}, "weight": 5},
```

Comment:
The Quant Bow looks a bit powerful here given how it's typically much harder to obtain (I see the materials to make it have a ZPM recipe). It's quite possible that with the way the combat content of the pack is balanced (which is to say poorly) it's actually fine here, but I'd rather be cautious for now.

Also the way I'm reading this, one out of every 6 level 4 weapons chest will contain a Quant Bow right now?